src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,34 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li
+            className={classNames({
+              'is-active': selectedTabId === tab.id,
+            })}
+            data-cy="Tab"
+            key={tab.id}
+          >
+            <a
+              href={`#${tab.id}`}
+              data-cy="TabLink"
+              onClick={() => {
+                if (selectedTabId !== tab.id) {
+                  onTabSelected(tab.id);
+                }
+              }}
```

Comment:
Move the onClick logic to a separate function to make code cleaner and more maintainable